### PR TITLE
start the branch for the Release 7 Notes.

### PR DESCRIPTION
**Pull request by fkr**

### CODE REVIEW

File: Release7.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+
+* Documentation for IaaS pentesting: https://docs.scs.community/docs/operating-scs/components/automated-pentesting-iaas/overview/
+* Documentation for KaaS pentesting: https://docs.scs.community/docs/operating-scs/components/automated-pentesting-kaas/overview/
+
+
+### Tech Preview: SCS Health Monitor
+
+As announced in the Release Notes for R6, a new health monitor was in the works during the last months. With R7 we are now at the state of releasing the first Tech Preview of the new SCS Health Monitor. It not only covers the infrastructure-as-a-service layer, but also has workflows for Kubernetes tests.
+
+While the SCSHM took many IaaS-related inspirations from the OSHM, the developer team for the new SCSHM has built an entirely new foundation for behavior-driven testing in SCS. The implementation is based on the Python `behave` framework, using the Gherkin language for test definitions. Using Python allows the tool to then use Python-native API cients for OpenStack and Kubernetes, with room for extension for many other tools in SCS as well.
```
Comment (gtema) [inline Release7.md]: cients => clients

File: Release7.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+* Repository: https://github.com/SovereignCloudStack/scs-health-monitor
+
+
+### Status Page: 1.0 Full Release
+
+The Status Page, [formerly available as a tech preview](https://scs.community/de/tech/2024/04/30/status-page-tech-preview-release/), has now been feature-complete for its first full release as version 1.0.
+
+The status page is the web frontend to our status API server, a record keeper of any incidents that may limit or break parts of your SCS infrastructure. While the tech preview version was limited to displaying this data, this full release contains not just improvements to the code working in the background, but also two new features:
+
+* The new *Maintenance Events* are a type of incident that are set in the future and allow you to announce planned times of limited or non-existant service well in advance, right on the status page.
```
Comment (gtema) [inline Release7.md]: non-existant => non-existent

File: Release7.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+
+As a user of a ClusterAPI based managed Kubernetes the interface is the `Cluster` object. To define this can be complicated and depends on the used ClusterClass. With [Cluster-Gen](https://github.com/SovereignCloudStack/cluster-gen/) the possible options will be pulled from an existing ClusterClass to provide a form which can be used to create the `Cluster` object to apply it to the management cluster, which results in the usable workload cluster.
+
+#### Tech Preview: Kamaji
+
+Kamaji is a Control Plane manager which makes it possible to host Control Planes on pods instead on dedicated VMs. This makes them highly scalable and safes a lot of resources. We now provide Cluster Stacks using Kamaji as Control Plane provider, which can be especially interesting for smaller CSPs.
+
+#### Openstack csp helper
+
+The [openstack-csp-helper](https://github.com/SovereignCloudStack/openstack-csp-helper) is a simple Helm Chart to build and deploy the needed secrets used in the mangement cluster (to create the machines) and in the workload cluster (to get persistent volumes and loadbalancers) from a usual `clouds.yaml` file.
```
Comment (gtema) [inline Release7.md]: mangement => management

File: Release7.md [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+
+### IAM
+
+The keycloak container has been updated and the build process has seen further automation.
+THe container is now deployed into the internal infrastructure k3s, using the modern
```
Comment (gtema) [inline Release7.md]: THe => The

**Review by gtema — CHANGES_REQUESTED** [review]
typos found

File: Release7.md (hunk 1 above)
Comment (garloff) [reply Release7.md]: Addressed in 362fb3c.

File: Release7.md (hunk 2 above)
Comment (garloff) [reply Release7.md]: Addressed in 362fb3c.

File: Release7.md (hunk 3 above)
Comment (garloff) [reply Release7.md]: Addressed in 362fb3c.

File: Release7.md (hunk 4 above)
Comment (garloff) [reply Release7.md]: Addressed in 362fb3c.

**Review by gtema — APPROVED** [review]

**Pull request merged**